test/integration/volume/attach_detach_test.go

```diff
@@ -246,6 +255,13 @@ func TestPodUpdateWithWithADC(t *testing.T) {
 	stopCh := make(chan struct{})
 	go informers.Core().V1().PersistentVolumeClaims().Informer().Run(stopCh)
 	go informers.Core().V1().PersistentVolumes().Informer().Run(stopCh)
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSIMigration) &&
```

Comment:
this could probably be refactored into a function that is shared by all of these tests

Reply:
Done